evap/evaluation/templates/evap_evaluation_edit_js.html

```diff
@@ -71,6 +71,20 @@
             });
         }
     }
+
+    // Update participant count displayed in form
+    let participant_list = document.getElementById("id_participants").tomselect;
+    participant_list.on("item_add", function() {
+        count_participants();
+    });
+    participant_list.on("item_remove", function() {
+        count_participants();
+    });
+
+    function count_participants(){
+        participant_counter = document.getElementById("participant_counter");
+        participant_counter.innerHTML = participant_list.items.length;
```

Comment:
I think we can remove some specifics from this: Let's make it so that the participant_counter does not need to have a `id` anymore and instead it has, say, `data-track-tomselect-count="#id_participants"`.

Additionally, we should use `innerText` instead of `innerHTML`